e2e/utils/zetacore.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	FungibleAdminName = "fungibleadmin"
+	AdminName = "fadmin"
```

Comment:
Same as my previous comment , we should keep this as it is

Reply:
58b2d923